sorc/cfs_prevmpi.fd/gblevn00.f

```diff
@@ -446,7 +446,7 @@ SUBROUTINE GBLEVENTS(IDATEP,IUNITF,IUNITE,IUNITP,IUNITS,SUBSET,
 
       COMMON /GBEVAA/ SID,OBS(15,255),QMS(12,255),BAK(12,255),XOB,
      $ YOB,DHR,TYP,NLEV
-      COMMON /GBEVBB/ PVCD,VTCD
+      COMMON /GBEVBB/ IPVCD,IVTCD
```

Comment:
Just noting: the `I` here is to make this an assumed integer type.  This is required now because the BUFR `UFBQCD` subroutine now takes an integer instead of a real status flag.